.travis.yml

```diff
@@ -6,10 +6,20 @@ node_js:
 - 8 # EOL: December 2019
 - 6 # EOL: April 2019
 
+os:
+  - linux
+
 matrix:
   fast_finish: true
   include:
-    - node_js: 8 # Version used to deploy to npm registry
+    - node_js: 8
+      os: windows
+      env: YARN_GPG=no # starts gpg-agent that never exits
+    - node_js: 8
+      os: osx
+    - name: "Production build"
+      node_js: 8 # Version used to deploy to npm registry
+      os: linux
       env: BUILD_ENV=production
```

Comment:
Do we need to update `deploy.on` a bit to target linux only?

Reply:
Production only runs on linux as can be seen from https://travis-ci.com/thelounge/thelounge/builds/88431681

And even then, `deploy.on` has no `os` filter, I already tried that.

Reply:
Sorry, I misread the config, thanks!